ledger/participant-integration-api/src/main/scala/platform/store/backend/oracle/OracleTable.scala

```diff
@@ -24,9 +24,8 @@ private[oracle] object OracleTable {
                     data(fieldIndex)(dataIndex),
                   )
                 }
-                preparedStatement.addBatch()
+                preparedStatement.execute()
```

Comment:
I propose to do one-by-one insertion due to the oracle bug with the batched insertion

Reply:
What is the performance impact on indexer benchmark running on oracle?

Reply:
Ignore, I now understand that this incarnation of the insert is only affecting package uplaods